workers/event_catcher/worker

```diff
@@ -5,8 +5,10 @@ require "bundler/inline"
 gemfile(false) do
   source "https://rubygems.org"
 
+  gem "activesupport", "~>6.1.6"
   gem "manageiq-loggers", "~>1.0"
   gem "manageiq-messaging", "~> 1.0"
+  gem "more_core_extensions"
```

Comment:
I'd like to get this out of here if possible in a future PR.  IMO, the server side should pre-process this before sending it in, so the worker doesn't have to know about settings layers or .to_i_with_method or any of that.  This would be simplified by not sending all of settings across the wire.